### ISSUE TITLE
About the data_test_file in Training process of imdb_classify_task

### ISSUE THREAD

**Issue by yanyan23333** [HiitLee/SALNet#2]
# classifier_imdb.py
## def main()
`dataset2 = TaskDataset(data_test_file, pipeline)`
`data_iter2 = DataLoader(dataset2, batch_size=cfg.batch_size, shuffle=False)`
`############`
`trainer = train.Trainer(cfg,
                                    dataName,
                                    stopNum,
                                    model1,
                                    model2,
                                    data_iter,
                                    data_iter2,
                                    data_iter3,
                                    data_iter_dev,
                                    torch.optim.Adam(model1.parameters(), lr=0.001),
                                    torch.optim.Adam(model2.parameters(), lr=0.005),
                                    get_device(), kkk + 1)`

**We don't seem to see how data_test_file is used in the training process in the paper (since one of  the advantages of your model is that there can be very little labeled_data), so what role does data_test_file ride in the training process? Thank you!**

**Comment by HiitLee** [comment]
Thank you for interest in my research. We use the data_test_file to check model performance per epoch. It has nothing to do with model training.
